.circleci/config.yml

```diff
@@ -435,9 +351,163 @@ commands:
 
                   ./scripts/submit-codecov-data.sh
 
-      # NOTE: We store the logs to ease with troubleshooting / debugging
-      - store_artifacts:
-          path: ~/artifacts
+            # NOTE: We store the logs to ease with troubleshooting / debugging
+            - store_artifacts:
+                path: ~/artifacts
+
+  init-cached-docker-image:
+    parameters:
```

Comment:
Please add description attribute here which describes what that command does / what kind of purposes it serves.